### ISSUE TITLE
Parameters in code and paper

### ISSUE THREAD

**Issue by JacobCummins** [Pointcept/PointTransformerV3#156]
Hi,

I've noticed that in your paper you have put that the learning rate for Indoor semantic segmentation settings (table 12). as 5e-3. In your code it is 0.006 (which is 6e-3). Do you know if you got your paper results from the parameters put in the paper or from the code?

Also for the S3DIS code, it has epochs set as 3000. If I wanted to reproduce your results, should I be using 3000 or 800 (from the paper appendix).

**Comment by Gofinge** [comment]
Hey, I think there is not much difference between these settings. Model is much more stable now. Also initized with pre-trained model weight like Sonata is highly recommended. It is super powerful.

**Comment by Garfield2005** [comment]
In the configuration file semseg-pt-v3m1-0-base.py, the parameter enc_depths is set to (2, 2, 2, 6, 2). However, in Table 15 of the PTV3 paper, it is listed as [2, 2, 6, 2].

Could you clarify why there is this discrepancy?
